node/rpc-api/static/full.json

```diff
@@ -0,0 +1,18680 @@
+{
```

Comment:
Do we have a way to pull this file from lotus? ie from github with a curl call? or is this just static and needs to be copy/pasted over

Reply:
No, they don't have a generated version in their repo, and not only that, their default make command doesn't produce a proper JSON file, so I had to modify it. This can be generated via the commands detailed in the original issue.

Reply:
Gotcha, that makes sense. Would be great if we could fetch it with a script to execute with the RPC schema generation

Reply:
Yeah, I mean, maybe, but build.rs runs every time you build forest. Fortunately, Rust is so fast it's barely noticeable, lol.